src/NuGet.Core/NuGet.Versioning/FloatRange.cs

```diff
@@ -202,17 +197,16 @@ public static bool TryParse(string versionString, out FloatRange range)
                         {
                             releasePrefix = actualVersion.Substring(versionString.LastIndexOf('-') + 1);
 
-                            if (actualVersion.EndsWith("-"))
+                            // For numeric labels 0 is the lowest. For alpha-numeric - is the lowest.
+                            if (releasePrefix.Length == 0 || actualVersion.EndsWith("."))
```

Comment:
I would say it is solved, it was probably due to speculation that a bug like that one fixed here could exist.